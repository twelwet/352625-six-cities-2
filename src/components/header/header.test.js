import React from "react";
import renderer from "react-test-renderer";
import Header from "./header.jsx";

jest.mock(`react-router-dom`, () => ({
  Link: () => null
}));

jest.mock(`../../components/user/user.jsx`, () => `User`);

it(`Header correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<Header />);

  expect(tree).toMatchSnapshot();
});
