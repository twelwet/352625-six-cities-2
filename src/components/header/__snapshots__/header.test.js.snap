// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer correctly renders after relaunch 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      />
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <User />
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
